src/fetch_receipts.py

```diff
@@ -0,0 +1,175 @@
+import os
+import re
+import sys
+from argparse import ArgumentParser, RawTextHelpFormatter
+from collections import namedtuple
+from urllib.error import HTTPError
+from urllib.request import urlretrieve
+
+from humanize import naturalsize
+import numpy as np
+import pandas as pd
+
+Receipt = namedtuple('Receipt', ['path', 'url'])
+
+
+class FetchReceipts:
+
+    BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+    DATA_DIR = os.path.join(BASE_DIR, 'data')
+    REGEX = r'^[\d-]{11}(current-year|last-year|previous-years).xz$'
+
+    def __init__(self, target, limit=None):
+        """
+        :param target: (str) path to the directory where images will be saved
+        :param limit: (int) limit the amount of files to save
+        """
+
+        # check if target directory exists
+        if not os.path.exists(target):
+            raise RuntimeError('Directory {} does not exist'.format(target))
+            sys.exit()
+
+        # check if target directory is a directory (not a file)
+        if not os.path.isdir(target):
+            raise RuntimeError('{} is a file, not a directory'.format(target))
+            sys.exit()
+
+        # dtype to be used by pandas (inside self.parse)
+        self.dtype = {
+            'document_id': np.str,
+            'congressperson_id': np.str,
+            'congressperson_document': np.str,
+            'term_id': np.str,
+            'cnpj_cpf': np.str,
+            'reimbursement_number': np.str
+        }
+
+        # save receipts
+        self.limit = limit
+        self.target = target
+        self.progress = {
+            'count': 0,
+            'size': 0,
+            'errors': list(),
+            'skipped': list()
+        }
+        for receipt in self.receipts:
```

Comment:
That sounds like a good idea, thanks!

Do you wanna step in and send a PR to this branch? These days my workload is focused on Issue #34. Otherwise I'l catch up here later.